tests/simpleclient.test.ts

```diff
@@ -0,0 +1,187 @@
+  //     const spotMarket = await Market.load(
+  //       connection,
+  //       new PublicKey(spotMarketAddress),
+  //       { skipPreflight: true, commitment: 'singleGossip' },
+  //       dexProgramId,
+  //     );
+  //     return { spotMarket, baseSymbol, quoteSymbol };
+  //   }
+  //
+  //   const mangoGroupTokenMappings = await createMangoGroupSymbolMappings(
+  //     connection,
+  //     mangoGroupIds,
+  //   );
+  //
+  //   // deposit usdc
+  //     await createTokenAccountWithBalance(
+  //       connection,
+  //       buyerOwner,
+  //       "USDC",
+  //       mangoGroupTokenMappings,
+  //       clusterIds.faucets,
+  //       100_000,
+  //     );
+  //   await performSingleDepositOrWithdrawal(
+  //     connection,
+  //     buyerOwner,
+  //     client,
+  //     mangoGroup,
+  //     mangoProgramId,
+  //     "USDC",
+  //     mangoGroupTokenMappings,
+  //     marginAccounts[0],
+  //     'deposit',
+  //     100_000,
+  //   );
+  //
+  //   // deposit base currency for each spot market
+  //   for (let i = 0; i < mangoGroupSpotMarkets.length; i++) {
+  //     const mangoGroupSpotMarket = mangoGroupSpotMarkets[i];
+  //     const { spotMarket, baseSymbol, quoteSymbol } =
+  //       await getSpotMarketDetails(mangoGroupSpotMarket);
+  //     const [orderSize, orderPrice, _] = await getOrderSizeAndPrice(
+  //       connection,
+  //       spotMarket,
+  //       mangoGroupTokenMappings,
+  //       baseSymbol,
+  //       quoteSymbol,
+  //       'buy',
+  //     );
+  //     const neededQuoteAmount = orderPrice * orderSize;
+  //     const neededQuoteAmountForAllTrades = neededQuoteAmount * 128;
+  //     await createTokenAccountWithBalance(
+  //       connection,
+  //       buyerOwner,
+  //       baseSymbol,
+  //       mangoGroupTokenMappings,
+  //       clusterIds.faucets,
+  //       neededQuoteAmountForAllTrades,
+  //     );
+  //
+  //     await performSingleDepositOrWithdrawal(
+  //       connection,
+  //       buyerOwner,
+  //       client,
+  //       mangoGroup,
+  //       mangoProgramId,
+  //       baseSymbol,
+  //       mangoGroupTokenMappings,
+  //       marginAccounts[0],
+  //       'deposit',
+  //       neededQuoteAmountForAllTrades,
+  //     );
+  //   }
+  // });
+
+  it('test place order', async () => {
+    const sc = await SimpleClient.create();
+    await sc.cancelOrders();
+    await sc.placeOrder(marketSymbol, 'buy', 0.0001, 10000);
+    await sc.placeOrder(marketSymbol, 'sell', 0.0001, 90000);
+    const ordersAfterPlacement = await sc.getOpenOrders(marketSymbol);
+    expect(ordersAfterPlacement.length).toBe(2);
+  });
+
+  it('test cancel a specific order', async () => {
+    const sc = await SimpleClient.create();
+    await sc.cancelOrders();
+    await sc.placeOrder(marketSymbol, 'buy', 0.0001, 48000);
+    const ordersAfterPlacement = await sc.getOpenOrders(marketSymbol);
+    await sc.cancelOrders(
+      marketSymbol,
+      ordersAfterPlacement[0].orderId.toString(),
```

Comment:
this feels clunky, the result of placeOrder should give you all you need to cancel the same order

Reply:
fixed and done